runbenchmark.py

```diff
@@ -47,6 +47,8 @@
                          "•auto: setup is executed only if strictly necessary. •skip: setup is skipped. •force: setup is always executed before the benchmark. •only: only setup is executed (no benchmark).")
 parser.add_argument('-k', '--keep-scores', type=str2bool, metavar='true|false', nargs='?', const=True, default=True,
                     help="Set to true [default] to save/add scores in output directory.")
+parser.add_argument('-e', action='store_true', dest="exit_on_error",
+                    help="If set, *any* task that does not complete with a model will cause the script to terminate.")
```

Comment:
```suggestion
parser.add_argument('-e', '--exit-on-error', action='store_true', dest="exit_on_error",
                    help="If set, the first task that does not complete with a model will cause the entire script to terminate.")
```
suggesting this to make it prettier when running 
```
python runbenchmark.py --help
```
Ideally, the long syntax should be enough to guess/remember/find an option.

also wondering if the `-k` option above still makes sense.

Reply:
I personally never used `-k` (I want the default behavior). Was there a specific reason for implementing it in the first place?

Reply:
Honestly, I can't remember, which is a good sign that it was not a good idea: I also always want the default behaviour.
We can get rid of it in a separate PR